### PR TITLE
Remove pypi-theme

### PR DESCRIPTION
**Pull request by di**
RIP.

### CODE REVIEW

**Review by ewjoachim — COMMENTED** [review]
https://warehouse.readthedocs.io/development/getting-started.html?highlight=logo#viewing-warehouse-in-a-browser
and
https://warehouse.readthedocs.io/development/patterns.html?highlight=logo#adding-new-dependencies

need to be updated :)

**Review by ewjoachim — APPROVED** [review]
+3 -43

Solid 👍 for me :D

**Comment by ewdurbin** [comment]
I think we need to pull over some of the images from https://github.com/python/pypi-theme/tree/master/pypi_theme/static/images, since this repo only holds placeholder SVG and PNG for the logo.

We may need to apply a different license to the directory holding them, since they are copyrighted to the PSF

File: warehouse/static/images/LICENSE [hunk 1]
```diff
@@ -0,0 +1,4 @@
+All images in this directory are copyrighted by the Python Software Foundation unless otherwise noted, and must be used in accordaince with the PSF's logo and trademark guidelines:
+
+    https://www.python.org/community/logos/
+    https://www.python.org/psf/trademarks/
```
Comment (di) [inline warehouse/static/images/LICENSE:4]: @ewdurbin Do you think this is sufficient?

File: warehouse/static/images/LICENSE [hunk 2]
```diff
@@ -0,0 +1,6 @@
+All images in this directory are copyrighted by the Python Software Foundation
+unless otherwise noted, and must be used in accordance with the PSF logo and
+trademark guidelines:
+
+    https://www.python.org/community/logos/
+    https://www.python.org/psf/trademarks/
```
Comment (di) [inline warehouse/static/images/LICENSE:6]: @ewdurbin Do you think this is sufficient?

Comment (ewdurbin) [reply warehouse/static/images/LICENSE:6]: I believe so

**Pull request merged**